svgscreen/template.svg

```diff
@@ -28,16 +28,51 @@ whitespace only tspan:s preventing underline to be properly rendered.
             font-variant-ligatures: none;
             dominant-baseline: central;
             white-space: pre;{{/* draw underline even when whitespace */}}
-            fill: {{$.ForegroundColor}};
+            fill: {{$.Foreground.Default}};
         }
+        .bg {
+            stroke-width: "0.5px";
+        }
+{{- if anyTrue $.Background.ANSIUsed}}
+        <!-- Background ANSI colors -->
+{{- end}}
+{{- range $k, $v := $.ANSIColors -}}
+        {{- if index $.Background.ANSIUsed $k}}
+        .ba{{$k}} { stroke: {{$v}}; fill: {{$v}}; }
+        {{- end}}
+{{- end}}
+{{- if anyTrue $.Foreground.ANSIUsed}}
+        <!-- Foreground ANSI colors -->
```

Comment:
Intension it that comment should end up in the output? otherwise can use `{{- /* ... */ -}}`

Reply:
yep, it's intentional, to be clearer what the classes are for. but maybe it's not necessary, WDYT?

Reply:
let's keep it!